lineman/app/components/thread_page/activity_card/activity_card.coffee

```diff
@@ -46,7 +46,7 @@ angular.module('loomioApp').directive 'activityCard', ->
 
     $scope.threadItemVisible = (item) ->
       addSequenceId(item.sequenceId)
-      $scope.discussion.markAsRead(item.sequenceId)
+      $scope.discussion.markAsRead(item.sequenceId) if AbilityService.isLoggedIn()
```

Comment:
I think I actually want to base this on whether the discussion model has a reader or not, so the activity card doesn't care if you're logged in or not.

```
# discussion_model.coffee
markAsRead: (sequenceId) ->
  return unless @discussionReaderId
```